zipkin-server/src/test/java/zipkin2/server/internal/eureka/BaseITZipkinEureka.java

```diff
@@ -60,7 +59,9 @@
     "zipkin.storage.type=", // cheat and test empty storage type
     "zipkin.collector.http.enabled=false",
     "zipkin.query.enabled=false",
-    "zipkin.ui.enabled=false"
+    "zipkin.ui.enabled=false",
+    // TODO: Remove when armeria doesn't set the vip address to include a port.
```

Comment:
actually, this is probably fine to leave here. I'll remove the TODO once I raise the PR in armeria to fix the default. It isn't really our job to test the default.

Reply:
raised the PR and removed the comment https://github.com/line/armeria/pull/5451